webpack.config.js

```diff
@@ -125,10 +128,10 @@ if ( TARGET_ENV === 'production' ) {
         },
       ]),
 
-      new webpack.optimize.OccurenceOrderPlugin(),
+      new webpack.optimize.OccurrenceOrderPlugin(),
 
       // extract CSS into a separate file
-      new ExtractTextPlugin( 'static/css/[name]-[hash].css', { allChunks: true } ),
+      new ExtractTextPlugin( 'static/css/[name]-[hash].css'),
```

Comment:
I'm not sure if plain filename is supported, nor why you removed `allChunks: true`, but according to [this](https://webpack.js.org/guides/migrating/#new-extracttextplugin-options-), the new code should be
```js
new ExtractTextPlugin({
  filename: 'static/css/[name]-[hash].css',
  allChunks: true
})